wikibot/wiki.py

```diff
@@ -1,59 +1,90 @@
 from hamper.interfaces import ChatCommandPlugin, Command
-import requests as re
-import json
+import requests
```

Comment:
Since there is a built in Python package named `re` (the regular expression library), `import requests as re` is bound to be confusing.